web/components/cycles/list/root.tsx

```diff
@@ -19,6 +24,27 @@ export const CyclesList: FC<ICyclesList> = observer((props) => {
           {cycleIds.map((cycleId) => (
             <CyclesListItem key={cycleId} cycleId={cycleId} workspaceSlug={workspaceSlug} projectId={projectId} />
           ))}
```

Comment:
This looks like it has been repeated, can you move it out to a separate component and reuse then in both the places